smart-contracts/contracts/burn_coins/src/state.rs

```diff
@@ -0,0 +1,58 @@
+use cosmwasm_std::Uint128;
+use cw_storage_plus::Map;
+
+pub const AMOUNT_BURNT: Map<String, Uint128> = Map::new("burnt_coins");
```

Comment:
Is that needed? What is the purpose of keeping track of burned coins if we can simply `query bank balances` over the contract's addy?

Reply:
coins would not be in the contract's addy since we call the bank modules burn